core/feed.ts

```diff
@@ -100,6 +100,16 @@ export async function changeFeed(
   feedId: string,
   changes: Partial<FeedValue>
 ): Promise<void> {
+  if (changes.reading) {
```

Comment:
We need to change post after changing the feed.

Imagine if we will add new post in the middle of this process.